sdk/workloads/Azure.ResourceManager.Workloads/tests/Tests/SviCRUDTests.cs

```diff
@@ -0,0 +1,216 @@
+            string rgName = "svi-loop-test-register-30Jun-avzone-44";
+
+            SubscriptionResource subscription = await Client.GetDefaultSubscriptionAsync();
+            var lro = await subscription.GetResourceGroups().GetAsync(rgName);
+            ResourceGroupResource rg = lro.Value;
+
+            // Stop operation
+            var sviObject1 = await lro.Value.GetSapVirtualInstanceAsync(sviName);
+            await sviObject1.Value.StopAsync(WaitUntil.Completed);
+            var sviObjectstatus = await lro.Value.GetSapVirtualInstanceAsync(sviName);
+            Assert.AreEqual(sviObjectstatus.Value.Data.Status, SapVirtualInstanceStatus.Offline);
+
+            // Start operation
+            await sviObject1.Value.StartAsync(WaitUntil.Completed);
+            sviObjectstatus = await lro.Value.GetSapVirtualInstanceAsync(sviName);
+            Assert.AreEqual(sviObjectstatus.Value.Data.Status, SapVirtualInstanceStatus.Running);
+        }
+
+        [TestCase]
+        [RecordedTest]
+        public async Task TestSVIPatchCall()
+        {
+            string sviName = "B83";
+            string rgName = "svi-loop-test-register-02Jul-singleserver-88";
+            string subID = "49d64d54-e966-4c46-a868-1999802b762c";
+
+            ResourceIdentifier sviResourceID = SapVirtualInstanceResource.CreateResourceIdentifier(subID, rgName, sviName);
+            ResourceIdentifier appServerResourceID =
+                SapApplicationServerInstanceResource.CreateResourceIdentifier(subID, rgName, sviName, "app0");
+            ResourceIdentifier dbServerResourceID =
+                SapDatabaseInstanceResource.CreateResourceIdentifier(subID, rgName, sviName, "db0");
+            ResourceIdentifier csServerResourceID =
+                SapCentralServerInstanceResource.CreateResourceIdentifier(subID, rgName, sviName, "cs0");
+
+            SapVirtualInstanceResource sapVirtualInstance = await Client.GetSapVirtualInstanceResource(sviResourceID).GetAsync();
+            SapDatabaseInstanceResource sapDatabaseInstanceResource =
+                await Client.GetSapDatabaseInstanceResource(dbServerResourceID).GetAsync();
+            SapApplicationServerInstanceResource sapApplicationServerInstanceResource =
+                await Client.GetSapApplicationServerInstanceResource(appServerResourceID).GetAsync();
+            SapCentralServerInstanceResource sapCentralServerInstanceResource =
+                await Client.GetSapCentralServerInstanceResource(csServerResourceID).GetAsync();
+
+            // Patch call for SVI and child resources
+            await sapVirtualInstance.AddTagAsync("TestKey1", "TestValue1");
+            await sapApplicationServerInstanceResource.AddTagAsync("TestKey1", "TestValue1");
+            await sapCentralServerInstanceResource.AddTagAsync("TestKey1", "TestValue1");
+            await sapDatabaseInstanceResource.AddTagAsync("TestKey1", "TestValue1");
+
+            Thread.Sleep(GetStatusIntervalinMillis);
```

Comment:
I'd like to know why we need this delay. Our test framework has time limit, this delay is likely to cause the test to time out.

Reply:
The delay was because when I fetched it immediately afterwards, the Get seemed to be failing. I added it as an extra safety measure.
It does not time out, I've run it multiple times.

Reply:
As discussed on call, it's not reduced to 2 seconds.